data/json/items/ammo/weldgas.json

```diff
@@ -15,6 +15,11 @@
     "color": "red",
     "ammo_type" : "weldgas",
     "effects": [ "COOKOFF" ],
-    "flags": [ "NO_DROP", "IRREMOVABLE" ]
+    "flags": [ "NO_DROP", "IRREMOVABLE" ],
+    "explode_in_fire" : true,
+    "explosion" : {
```

Comment:
Not necessarily
At the moment it is true, but that's only because things like chain explosions aren't in and because all `explosion_iuse` tools specify their own explosions inside the actor. There are objects that should only explode under specific conditions, such as C4 and ANFO. They could explode in raging fire, but not in density 1 one.